api/index.md

```diff
@@ -9,27 +9,24 @@ has_children: true
 The [Data Commons Graph](https://datacommons.org) aggregates data from many
 different [data sources](https://datacommons.org/datasets) into a single
 database. Data Commons is based on the data model used by
-[schema.org](https://schema.org), for more information see [our guide to the data model](/data_model.html).
+[schema.org](https://schema.org); for more information, see [our guide to the data model](/data_model.html).
 
-The **Data Commons API** is a set of APIs that allow developers to
+The **Data Commons API** allows developers to
 programmatically access the data in Data Commons.
 Data Commons provides APIs in each of the different views
 (graph, timeline, etc.) for use in different contexts. More specifically, we provide
-1. **REST** APIs that can be used by javascript based applications
-1. **Pandas** APIs for use in Python notebooks
-1. **Google Sheets** APIs for use in Google Sheets
-1. **Python** APIs
-1. **SQL** (Coming soon)
```

Comment:
Per https://developers.google.com/style/future, removing future feature documentation. (If needed, I can write up a blog post for roadmap if that would be helpful.)